packages/api/src/controllers/clip.ts

```diff
@@ -103,9 +109,57 @@ app.post("/", validatePost("clip-payload"), async (req, res) => {
   );
 
   res.json({
-    task,
+    task: { id: task.id },
     asset,
   });
 });
 
+async function getRunningRecording(content: DBStream, req: Request) {
+  let objectStoreId: string;
+
+  const session = await db.stream.getLastSessionFromSessionsTable(content.id);
```

Comment:
I've sent a PR to fix this query, please merge it into this PR (base branch is already set, just review/make any changes/merge it)

https://github.com/livepeer/studio/pull/1896